testcases/OpTestEEH.py

```diff
@@ -42,6 +42,14 @@
 from common.OpTestUtil import OpTestUtil
 from common.OpTestSystem import OpSystemState
 
+class EEHRecoveryFailed(Exception):
+    def __init__(self, thing, dev, log=None):
+        self.thing = thing
+        self.what = what
```

Comment:
This is failing with below error.

----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/pridhivi/refactor/op-test-framework/testcases/OpTestEEH.py", line 314, in runTest
    raise EEHRecoveryFailed("PHB domain", domain)
  File "/root/pridhivi/refactor/op-test-framework/testcases/OpTestEEH.py", line 48, in __init__
    self.what = what
NameError: global name 'what' is not defined

----------------------------------------------------------------------
Ran 1 test in 163.077s

FAILED (errors=1)

Run basic frozen PE testcase to reproduce this.